pkg/lint/fileparser.go

```diff
@@ -0,0 +1,287 @@
+package lint
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"regexp"
+	"strconv"
+	"strings"
+	"text/template/parse"
+
+	"github.com/Masterminds/sprig"
+	"github.com/ghodss/yaml"
+
+	"github.com/giantswarm/microerror"
+	pathmodifier "github.com/giantswarm/valuemodifier/path"
+)
+
+var (
+	fMap                 = dummyFuncMap()
+	includes             = &includeExtract{[]string{}}
+	templatePathPattern  = regexp.MustCompile(`(\.[a-zA-Z].[a-zA-Z0-9_\.]+)`)
+	yamlErrorLinePattern = regexp.MustCompile(`yaml: line (\d+)`)
+)
+
+func init() {
+	fMap["include"] = includes.include
+}
+
```

Comment:
Why fields of `ValueFile` are private but fields of `ValuePath` are public?

Reply:
Changed them to private.